package.json

```diff
@@ -33,6 +33,8 @@
     "@babel/preset-typescript": "^7.14.5",
```

Comment:
Curious does the test command run the playwright tests? or can we add a playwright one? Just so npm run tests ends up actually running both playwright and npm tests.

Reply:
Nah, it just runs the jest tests. To run the playwright ones we would need to run `npx playwright test`. The suite of tests provided by the init script are the following

```

  npx playwright test
    Runs the end-to-end tests.

  npx playwright test --ui
    Starts the interactive UI mode.

  npx playwright test --project=chromium
    Runs the tests only on Desktop Chrome.

  npx playwright test example
    Runs the tests in a specific file.

  npx playwright test --debug
    Runs the tests in debug mode.

  npx playwright codegen

Reply:
As discussed offline, the test command runs both jest and playwright. Actions that currently call npm run test should be changed to just be running the unit tests

Reply:
As discussed offline, realized that we can pass down inputs to the reusable workflows. 

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idwith